arm-insights/2016-03-01/swagger/insightsManagementClient_LogProfiles.json

```diff
@@ -64,21 +64,32 @@
             },
             "examples": {
               "application/json": {
-                "storageAccountId": "/subscriptions/6983c752-c9b8-48dd-b4d4-da739beb7e98/resourceGroups/Default-Storage-WestUS/providers/Microsoft.ClassicStorage/storageAccounts/armlogs1",
-                "serviceBusRuleId": "/subscriptions/6983c752-c9b8-48dd-b4d4-da739beb7e98/resourceGroups/Default-ServiceBus-WestUS/providers/Microsoft.ServiceBus/namespaces/myNamespace/RootManageSharedAccessKey",
-                "locations": [
-                  "eastus",
-                  "westus"
-                ],
-                "categories": [
-                  "Write",
-                  "Delete",
-                  "Action"
-                ],
-                "retentionPolicy": {
-                  "enabled": true,
-                  "days": 1
-                }
+                "value": [
+                  {
+                    "id":"/subscriptions/df602c9c-7aa0-407d-a6fb-eb20c8bd1192/providers/microsoft.insights/logprofiles/default",
+                    "type":null,
```

Comment:
I cannot tell why the service responds with null here and in the location. This is Vinicius area.  
Here I am just reflecting the actual behavior of the services.